### PR TITLE
Course cleaner

### PR DESCRIPTION
**Pull request by fraser-mcleod**
This code adds the following important functionality:
- Creates DB specifically for prerequisite strings which makes it easy to work with for parsing and converting (rather than scrubbing every time)
- courseForms.py has the functionality to separate and transform prerequisite forms in various ways.

### CODE REVIEW

File: server/api/models.py [hunk 1]
```diff
@@ -30,4 +30,14 @@ class Dependency(models.Model):
...
-        ]
\ No newline at end of file
+        ]
+
+class CoursePreqInfo(models.Model):
+    course_code = models.CharField(max_length=10, null=True)
+    name = models.CharField(max_length=255, null=True)
+    prereqString = models.TextField(null=True)
+
+class TestDB(models.Model):
```
Comment (marmikpatel19) [inline server/api/models.py:40]: If this is for testing, can we get rid of it?

**Review by marmikpatel19 — CHANGES_REQUESTED** [review]
Can we delete all the code concerning testing? We probably don't want to push testing code into main. Is there a purpose for it?

**Comment by fraser-mcleod** [comment]
> Can we delete all the code concerning testing? We probably don't want to push testing code into main. Is there a purpose for it?

We can delete it, however, it will be helpful for Kyle and Ini when they set up locally to test their DB. Otherwise, they would have to web scrape/edit existing post functions/post into a real DB to test if their connection is working.

**Comment by marmikpatel19** [comment]
Ah, got it. Sounds good then!

**Review by marmikpatel19 — APPROVED** [review]

**Pull request merged**